ereuse_tag/definition.py

```diff
@@ -129,3 +129,29 @@ def import_tags(self, csv: Path):
                 max_id = max(max_id, cls.decode(id))
         db.session.execute('ALTER SEQUENCE tag_id RESTART WITH {}'.format(max_id + 1))
         db.session.commit()
+
+
+class VersionDef(Resource):
+    __type__ = 'Version'
+    SCHEMA = None
+    VIEW = None  # We do not want to create default / documents endpoint
+    AUTH = False
+
+    def __init__(self, app,
+                 import_name=__name__,
+                 static_folder=None,
+                 static_url_path=None,
+                 template_folder=None,
+                 url_prefix=None,
+                 subdomain=None,
+                 url_defaults=None,
+                 root_path=None,
+                 cli_commands: Iterable[Tuple[Callable, str or None]] = tuple()):
+        super().__init__(app, import_name, static_folder, static_url_path, template_folder,
+                         url_prefix, subdomain, url_defaults, root_path, cli_commands)
+
+        d = {'ereuse_tag': '0.0.1a1'}
```

Comment:
Is that hardcoded version value?

Reply:
The view has a parameter 'default' and this try to put something in this default. I changed this and put 0.0.0 of default version.

Reply:
What about renaming variable `d --> defaults` ?

Reply:
Done